usage.md

```diff
@@ -3,3 +3,99 @@
 1. [Use the client to test your server with a companion specification instance (Windows)](doc/usage_as_model_test.md)
 2. [Use the client as a gateway to connect your server to the umati.app (Windows)](doc/usage_for_dashboard.md)
```

Comment:
We can add the process to get the credentials for someone who passed the local test and to validate with umati infrastructure, he/she can contact XYZ@ABC.com to get the required information.

Reply:
Extended